Homework №3 from 21.09.2022/task2.c

```diff
@@ -0,0 +1,149 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#define MAXNUM 100
+#define lenArrForTest 9
```

Comment:
```suggestion
#define LENGTH_TEST_ARRAY 9
```
Единообразие должно быть и в define именах.